scripts/release.js

```diff
@@ -178,7 +214,23 @@ if (release_option === '5') {
 } else {
   spawnOrFail('git', ['push origin HEAD:release -f']);
 }
-process.chdir(path.join(__dirname, '../demo/meeting/serverless'));
-logger.log("Deploying unique release candidate Meeting Demo URL...");
-const formattedVersion = versionString.replace(/\./g, "-");
-spawnOrFail('node', [`./deploy.js -r us-east-1 -b chime-sdk-components-demo-${formattedVersion} -s chime-sdk-components-demo-${formattedVersion}`]);
+
+// Pack the latest version of React library and install it in meeting demo
+process.chdir(path.join(__dirname, '..'));
+spawnOrFail('npm', ['run build']);
+spawnOrFail('npm', ['pack']);
+process.chdir(path.join(__dirname, '../../amazon-chime-sdk/apps/meeting'));
+spawnOrFail('npm', [
+  `install ../../../amazon-chime-sdk-component-library-react/amazon-chime-sdk-component-library-react-${versionString}.tgz`,
```

Comment:
Need this in deploy script too.

Reply:
Yes, both of them are pack and then install. But the directory is not the same, may need to pass it as parameter